test/core/security/credentials_test.cc

```diff
@@ -56,24 +56,28 @@
 #include "src/core/lib/security/credentials/google_default/google_default_credentials.h"
 #include "src/core/lib/security/credentials/jwt/jwt_credentials.h"
 #include "src/core/lib/security/credentials/oauth2/oauth2_credentials.h"
+#include "src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h"
+#include "src/core/lib/security/credentials/xds/xds_credentials.h"
 #include "src/core/lib/security/transport/auth_filters.h"
 #include "src/core/lib/uri/uri_parser.h"
 #include "test/core/util/test_config.h"
 
-using grpc_core::internal::grpc_flush_cached_google_default_credentials;
-using grpc_core::internal::set_gce_tenancy_checker_for_testing;
+namespace grpc_core {
```

Comment:
The namespace changes here are definitely desirable, but they should probably go in a separate PR.